myia/py_implementations.py

```diff
@@ -0,0 +1,144 @@
+"""Implementations for the debug VM."""
+
+
+from typing import Dict, Callable
+from copy import copy
+from myia import primops
+
+
+implementations: Dict[primops.Primitive, Callable] = {}
+
+
+def make_registrar(store):
```

Comment:
Maybe move this to `utils`?

Reply:
Sure. I think I might make this a class, actually.